analysis/run_score_function_experiments.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+START_SEED=456
+NUM_SEEDS=10
+CYCLES=100
+FILE="analysis/submit.py"
+
+for SEED in $(seq $START_SEED $((NUM_SEEDS+START_SEED-1))); do
+    # section kid
+    python $FILE --env cover --approach interactive_learning --seed $SEED --excluded_predicates Covers --interactive_query_policy nonstrict_best_seen --interactive_score_function trivial --num_online_learning_cycles $CYCLES
+    # ours
+    python $FILE --env cover --approach interactive_learning --seed $SEED --excluded_predicates Covers --interactive_query_policy threshold --interactive_score_threshold 0.5 --interactive_score_function entropy --num_online_learning_cycles $CYCLES
+    # silent kid
```

Comment:
a truly silent kid would never query about anything, but this one may query one or a few times, right? can we change this to be truly silent, to make the eventual explanation of this baseline simpler?

also just to check -- this silent kid will still be learning things and changing over the course of online interactions, even though it's not querying, because we're adding all of the collected data into the set that's used for NSRT learning. right?

Reply:
yep that makes sense, i'll make the silent kid truly silent :P

and yes, that's right